### DISCUSSION TITLE
MusicBrainz Implementation

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23022] [Ideas] [answered]
Hi there! I have been playing osu! since 2015, and have been an avid player of the game since. I have been very impressed with the direction lazer has been going, and would like to suggest an idea that's popped in my head lately.

Osu! has really influenced my music taste over the years. When I first played the game I was introduced to Lost by Chasers, and that really started getting me interested in electronic music. I find it funny now as music artists I've come across nowadays on my own have since had their music added to the Featured Artists program such as the Hardcore Utopia crew and Geoxor.

My friends and I in a Discord server have music nights, and a lot of music I enjoy I find through Osu!. I've also recently been getting involved in Open Database projects, and I recently had this thought that it would be really cool if Osu! had a feature to look at the MusicBrainz profiles of music artists.

I don't fully know how it would be implemented, but I was thinking it could either be added as an outlink in the dropdown menu here, or somewhere on the in-game Details page
![image](https://user-images.githubusercontent.com/86493210/227998831-f5082281-cecc-46be-a002-40a504a57723.png)

I feel this would be a great idea for the game, as players are often exposed to new music they enjoy while playing it, and having the link between the game and the MusicBrainz database I feel would help expose them to the rest of a music artist's catalogue, therefore giving more discoverability and support to said music artists. That in turn may also come full circle with mappers having a more direct means to discover new songs to map.

MusicBrainz's core data is CC0, basically making the data within it Public Domain, which aligns with the Open-Source aspect of lazer. While I am unsure what licensing issues may arise from this kind of functionality being in game, an outline of the licenses MusicBrainz utilizes can be found here: https://musicbrainz.org/doc/About/Data_License

Thanks in advance for taking the time to look at my suggestion, and I hope that by no means I'm wasting people's time with it.

**Comment by peppy** [comment, marked as answer]
We've generally kept osu! a very closed system, never relying on external data sources so this would need some consideration from that angle. I'm not strongly against it, but I'm also not sure how much benefit it would add to the game.

**Reply by TheFearlessDeath500** [reply]
it wouldn’t benefit that much imo, it feels more like a 3rd-party plugin that someone would use
